### PR TITLE
feat/added payment intents type for customer and invoice objects

### PR DESCRIPTION
**Pull request by ejkkan**
### Description

This PR adds [Stripe payment intents](https://stripe.com/docs/api/payment_intents/object) to the stripe package.
It adds the ability to query payment intent on the cutomer and invoice objects.

Example query 
```ts
query ($id: String!){
  stripe {
    customer(id: $id) {
     id
      paymentIntents {
        data
        {
          status
        }
      }
      invoices {
        data {
          paymentIntent {
            id
            invoice {
              paid
            }
          }
        }
      }
    }
  }
}
```

### CODE REVIEW

File: packages/stripe-graphql-js/src/schema/payment-intent.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        nullable:true
+    }),
+    canceled_at: t.exposeInt('canceled_at', {
+        nullable: true
+    }),
+    transfer_group: t.exposeString('transfer_group',{ 
+      nullable:true
+    }),
+    cancellation_reason: t.exposeString('cancellation_reason', {
+        nullable:true
```
Comment (elitan) [inline packages/stripe-graphql-js/src/schema/payment-intent.ts:48]: Please make the field names camelCase.

`payment_method_types` => `paymentMetohdTypes`. Etc.

**Review by elitan — CHANGES_REQUESTED** [review]
Thanks for this PR!

File: packages/stripe-graphql-js/src/schema/payment-intent.ts (hunk 1 above)
Comment (ejkkan) [reply packages/stripe-graphql-js/src/schema/payment-intent.ts]: Ah, my bad!
Sorry for the late reply, it should be fixed now :)

**Comment by ejkkan** [comment]
I messed up and continued on the work from this branch with [other contributions](https://github.com/nhost/nhost/pull/1036).
I'll close this in favour for the other one that contains the same stuff including some more stuff as well.

**Pull request closed without merging**